cms/templates/logo.pt

```diff
@@ -0,0 +1,9 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal"
+     metal:define-macro="logo">
+    <div id="logo-container" tal:define="localisation view.get_localisation()" tal:condition="localisation">
+        <a href="/" tal:condition="localisation.logo_image">
+            <img src="${view.get_image_url(localisation.logo_image_host, localisation.logo_image)}" border="0" alt="${localisation.logo_description}"/>
+        </a>
+        <div tal:condition="localisation.logo_text">${localisation.logo_text}</div>
```

Comment:
Ok. How about `default_image_src` argument for the macro? If it's left out, nothing is rendered. Apps can then specify `default_image_src="/static/img/logo.png"` if they want.